client/src/components/NavLinkDropdownMenu/NavLinkDropdownMenu.js

```diff
@@ -5,7 +5,7 @@ import "./NavLinkDropdownMenu.css";
 const NavLinkDropdownMenu = (props) => (
   <ul className='NavLinkDropdownMenu'>
     {props.menuList.map(li => 
-      <Link to={li.to}>
+      <Link to={li.to} key={li.text}>
         <li>{li.text}</li>
       </Link>
     )}
```

Comment:
This should probably be fixed on the branch for this component, rather than in this PR

Reply:
It's fine here - going forward though, you want to keep your PRs relatively atomic (only focused on the change you're requesting be merged)